sos/report/plugins/doca_xplane.py

```
@@ -0,0 +1,71 @@
+        )
+
+        res = self.collect_cmd_output(f"{CLIENT_COMMAND} get-status")
+
+        if res["status"] != 0:
+            self._log_error("Failed to get status")
+            return
+
+        try:
+            status = json.loads(res["output"])
+            num_planes = status["topologySummary"]["planes"]
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)